### ISSUE TITLE
Extra Credit Assignment and Grades

### ISSUE THREAD

**Issue by JinghongLiuMT** [Media-Ed-Online/intro-web-dev-2018spr-resources#78]
Hello guys, I am trying to do the "Site Analytic" extra credit assignment on Moodle, but I don't know why the website shows page not found when I try to open it.

And also I do not know why there are only one assignment and one project are graded on Moodle, it let me feel confused about which assignment should be revised.

**Comment by Caitlynkay** [comment]
Same! I've emailed our instructors a couple times but haven't heard anything. I would like to know my grades so I can revise them if I need to, too.

**Comment by justmolliecate** [comment]
Me too. I have no idea when our last assignment is due or what the deal is with extra credit. Has anyone had contact with her about it?

**Comment by justmolliecate** [comment]
@shevin1992 @goodrichnic @JustineEvansUM

**Comment by Drakklin** [comment]
Im pretty much concerned about this entire class at this time....really worrying.

**Comment by Jorpatterson** [comment]
I agree. I have tried emailing and have heard no response. I have never had a class where I have no idea what my grade is. If we unknowingly did something wrong in the beginning and have continued to do it wrong will all consecutive assignments be wrong? Additionally I guess, since a few assignments were scratched then are all the other assignments worth more?

**Comment by chatga** [comment]
I am also worried about my grade. I’ve emeailed her as well with no response

**Comment by Xueyanzz** [comment]
me too

**Comment by Caitlynkay** [comment]
If anyone hears from anyone please keep us updated!

**Comment by JinghongLiuMT** [comment]
Hi guys, please check your school email, there is one email that sent this afternoon from our professor. It may help you to answer your questions.
